abstractions/php/composer.json

```diff
@@ -3,6 +3,7 @@
     "description": "Abstractions for Kiota",
     "type": "library",
     "license": "MIT",
+    "version": "1.0.28",
```

Comment:
@baywet What informed this version number?
Also since we're not publishing PHP packages yet, would this be necessary?
Currently [Serialization](https://github.com/microsoft/kiota/blob/main/serialization/php/json/composer.json#L25) and [Request Adapter](https://github.com/microsoft/kiota/blob/main/http/php/guzzle/composer.json#L16) rely on branch versions

Reply:
Oops too late lol. Can work on it on another PR

Reply:
ah, sorry, I thought it was overlooked and was missing. This specific version number is just aligned with ts/java/dotnet for simplicity for us.

Reply:
No worries, the auth PR for PHP can sort that out.

Reply:
yep, just rebase and then run `git revert 73609bc3b80708e9b9cfbcc720f65d7741b49dd0`